### PR TITLE
Add label for spyglass changes

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/cc @paulangton 
/assign @BenTheElder @cjwagner @fejta

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
Would love to see a tracking issue for spyglass roadmap, this will help aggregate changes in one label query.

File: prow/deck/OWNERS [hunk 1]
```diff
@@ -1,2 +1,2 @@
 labels:
- - area/prow/deck
+ - area/prow/deck
```
Comment (BenTheElder) [inline prow/deck/OWNERS]: nit: newline? /shrug

**Comment by paulangton** [comment]
Looks good to me as well, I'll work on that tracking issue 👍

File: prow/deck/OWNERS (hunk 1 above)
Comment (stevekuznetsov) [reply prow/deck/OWNERS]: not worth bothering for a re-LGTM ;)

Comment (BenTheElder) [reply prow/deck/OWNERS]: I'm quick to re-lgtm generally, and flip-flopping newlines is 👎 
unix files or bust! 😛

**Pull request merged**